src/spatialdata/_io/_utils.py

```diff
@@ -282,6 +288,58 @@ def save_transformations(sdata: SpatialData) -> None:
     """
     from spatialdata.transformations import get_transformation, set_transformation
 
-    for element in sdata._gen_elements_values():
+    for element in sdata._gen_spatial_element_values():
         transformations = get_transformation(element, get_all=True)
         set_transformation(element, transformations, set_all=True, write_to_sdata=sdata)
+
+
+def read_table_and_validate(
+    zarr_store_path: str, group: zarr.Group, subgroup: zarr.Group, tables: dict[str, AnnData]
+) -> dict[str, AnnData]:
+    """
+    Read in tables in the tables Zarr.group of a SpatialData Zarr store.
+
+    Parameters
+    ----------
+    zarr_store_path : str
```

Comment:
types are not needed in the docstring because they are inferred by Sphynx, I will remove them now in a commit.

Reply:
agree all types should be removed across docstrings

Reply:
I think all type hints are now removed from doc strings.